databricks/koalas/series.py

```diff
@@ -2201,6 +2201,45 @@ def transform(self, func, *args, **kwargs):
         else:
             return self.apply(func, args=args, **kwargs)
 
+    def round(self, decimals=0):
+        """
+        Round each value in a Series to the given number of decimals.
+
+        Parameters
+        ----------
+        decimals : int
+            Number of decimal places to round to (default: 0).
+            If decimals is negative, it specifies the number of
+            positions to the left of the decimal point.
+
+        Returns
+        -------
+        Series object
+
+        See Also
+        --------
+        DataFrame.round
+
+        Examples
+        --------
+        >>> df = ks.Series([0.028208, 0.038683, 0.877076], name='x')
+        >>> df
+        0    0.028208
+        1    0.038683
+        2    0.877076
+        Name: x, dtype: float64
+
+        >>> df.round(2)
+        0    0.03
+        1    0.04
+        2    0.88
+        Name: x, dtype: float64
+        """
+        if isinstance(decimals, int):
+            return _col(self.to_dataframe().round(decimals=decimals))
+        else:
+            raise ValueError("decimals must be an integer")
```

Comment:
@RainFung, can we do `F.round` here? Once new DataFrame is created, it causes an issue such as https://github.com/databricks/koalas/issues/405. We can avoid this via not creating new DataFrame via `to_dataframe()~.

Reply:
Is there a need update functions which contains to_dataframe() ?